tests/config_test_ipu_dataloader_multitask.yaml

```diff
@@ -0,0 +1,346 @@
+# Testing the gcn model with the PCQMv2 dataset on IPU.
+constants:
+  name: &name neurips2023_small_data_gcn
```

Comment:
This config is just data-loading right? I assume there is not a way to run the full model on IPU without physical IPU?

Reply:
Those tests were also compiling the model and starting training, which while okay on a machine with IPUs, not so good on the runner. So I'm skipping the "data loader" tests for now, while we get the IPU model working with lightning. (In a different PR)